extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetTableReadWriteTest.java

```diff
@@ -607,6 +612,41 @@ private interface TestParquetTableWriter {
     TestParquetTableWriter multiWriter = (table, destFile) -> ParquetTools.writeTables(new Table[] {table},
             table.getDefinition(), new File[] {destFile});
 
+    /**
```

Comment:
Have you tested backwards compatibility?

Reply:
IIRC I did. Will add a unit test for that case.